init/CMakeLists.txt

```diff
@@ -77,6 +77,9 @@ target_link_libraries(hpx_init PUBLIC hpx)
 target_link_libraries(hpx_init PRIVATE hpx_private_flags)
 target_link_libraries(hpx_init PRIVATE hpx::boost)
 target_compile_definitions(hpx_init PRIVATE HPX_APPLICATION_EXPORTS)
+if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
+  target_compile_options(hpx_init PRIVATE -U_ZN9hpx_start20app_name_libhpx_wrapE)
+endif()
```

Comment:
Uhh, really? A mangled name in the build system? This doesn't look right to me.

Reply:
Any suggestion ? I'm out of ideas..